blip-tap.md

```diff
@@ -924,28 +924,56 @@ accept if favorable
 
 ##### Request For Quote (`tap_rfq`)
 
-When a receiver wishes to receive `N` units of TAP asset ID `asset_id`, a new
-p2p message `tap_rfq` is sent with the following structure:
+In this section, we will describe the message type a node uses to request a
+quote from a channel counterparty. When the requesting node wants to obtain
```

Comment:
> obtain (or send)

?

Reply:
I think it'd also be useful to add an example below. Eg:
  * In the LN scenario, when a party has asset `X`, and wants to pay an invoice, what should be set for `in_asset_id` and `out_asset_id`. 
  * Same scenario above, but when they want to receive asset `X`.